clog/loggers.py

```diff
@@ -303,6 +303,8 @@ def __init__(self):
         self.lines = {}
 
     def log_line(self, stream, line):
+        assert isinstance(stream, (bytes, six.text_type))
+        assert isinstance(line, (bytes, six.text_type))
```

Comment:
Yeah, the latter is probably best. Don't think people will be needing the hex for debugging purposes, and definitely looks cleaner